TYPO3.Flow/Classes/TYPO3/Flow/Cache/CacheManager.php

```diff
@@ -358,7 +358,7 @@ protected function flushConfigurationCachesByChangedFiles(array $changedFiles)
         $objectConfigurationCache = $this->getCache('Flow_Object_Configuration');
         $caches = array(
             '/Policy\.yaml/' => array('Flow_Security_Authorization_Privilege_Method', 'Flow_Persistence_Doctrine', 'Flow_Persistence_Doctrine_Results', 'Flow_Aop_RuntimeExpressions'),
-            '/Routes([^\/]*)\.yaml/' => array('Flow_Mvc_Routing_Route', 'Flow_Mvc_Routing_Resolve'),
+            '/Routes|Settings([^\/]*)\.yaml/' => array('Flow_Mvc_Routing_Route', 'Flow_Mvc_Routing_Resolve'),
```

Comment:
Yeah, but the settings is just an assoc array at that point, no? So a simple recursive array_diff_assoc could reveal the changed settings. At least worth a second thought IMHO - as said, it could maybe be a nice general feature.